src/utils.js

```diff
@@ -711,5 +711,44 @@ go.utils = {
         return choices;
     },
 
+    // function used to validate months for states 5A/5B
+    is_valid_month_last_period: function(today, periodLastYear, choiceValue) {
+        if (periodLastYear) { today.subtract('year', 1); }     // substract year if last period happened the previous year
```

Comment:
Even though there's no Pep8 for javascript that determines the max amount of characters per line, if you go over a certain amount like here it gets hard to review. So rather put the comment on the line above in this instance.